docs/guides.md

```diff
@@ -1118,29 +1135,33 @@ varstore-get <vm-uuid> <guid> <name> | hexdump -Cv
 
 The GUID and name for varstore-get are the values returned by `varstore-ls`.
 
-#### Remove Certs from the Pool
+### Remove Certs from the Pool
 
 To remove the installed certs in the pool:
 
 ```
 secureboot-certs clear
 ```
 
-Note that this does not remove the certs from the VM or from disk. In order to clear the certs from the VM it is required to use `varstore-rm`. See [Remove Certs from the VM](#remove-certs-from-the-VM) on each VM and also remove the ".auth" files for the certs you'd like to remove (found in `/var/lib/uefistored/` and `/usr/share/uefistored/`).
+Note that this does not remove the certs from the VMs or from disk. In order to clear the certs from the VMs it is required to use `varstore-rm`. See [Remove Certs from a VM](#remove-certs-from-a-VM) on each VM and also remove the ".auth" files for the certs you'd like to remove (found in `/var/lib/uefistored/` and `/usr/share/uefistored/`).
 
-#### Remove Certs from the VM
+### Remove Certs from a VM
 
 To remove a cert use `varstore-rm`.
 
-For example, to remove the `db` from a VM.
+For example, to remove the `dbx` from a VM.
 
 ```
-varstore-rm <vm-uuid> d719b2cb-3d3a-4596-a3bc-dad00e67656f db
+varstore-rm <vm-uuid> d719b2cb-3d3a-4596-a3bc-dad00e67656f dbx
 ```
 
 Note that the GUID may be found by using `varstore-ls <vm-uuid>`.
 
-#### How XCP-ng Manages the Certificates
+:::tip
+Any certificate removed from the VM but still present in the pool configuration or on the host's disk at `/var/lib/uefistored/` will be automatically added back the next time the VM starts.
```

Comment:
True, except for the case when a lower privileged certificate is missing from the file system.

Given db < dbx < KEK < PK, if cert x < y and x is not in `/var/lib/uefistored/`, then y will not be loaded upon VM start.